bin/ColorScript/Sources/ColorScriptCore/Sources/ColorScript.swift

```diff
@@ -0,0 +1,34 @@
+// swiftlint:disable force_try
+import Foundation
+
+public enum ColorScriptError: Error {
+  case missingArguments
+}
+
+public final class ColorScript {
+  private let arguments: [String]
+
+  public let inPath = "Resources/Colors.json"
+  public let outPath = "Resources/Colors.swift"
+
+  public init(arguments: [String] = CommandLine.arguments) {
+    self.arguments = arguments
+  }
+
+  public func run() throws {
+
+    print(inPath)
+    print(outPath)
+
+    let data = try! Data(contentsOf: URL(fileURLWithPath: inPath))
```

Comment:
I think we should avoid using `try!` since we now have a nice way to fail gracefully. We could wrap this in a `do catch` and throw a `ColorScriptError` with a descriptive error message